WooCommerce/Classes/POS/Models/PointOfSaleOrderTotals.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+struct PointOfSaleOrderTotals: Equatable {
+    // Arguably these should be unformatted, and then we can rely on the SwiftUI formatter.
+    // To do that, we'd need to include Decimal amounts and the order currency in this struct.
```

Comment:
FYI – generally I'd expect formatting to be a view concern, but with our fiddly rules I didn't want to change it in this PR.